code/game/objects/items/singularityhammer.dm

```diff
@@ -72,14 +76,15 @@
 	if(!proximity)
 		return
 	if(wielded)
-		if(charged == 5)
-			charged = 0
+		if(charged == TRUE)
+			charged = FALSE
 			if(istype(A, /mob/living/))
 				var/mob/living/Z = A
 				Z.take_bodypart_damage(20,0)
 			playsound(user, 'sound/weapons/marauder.ogg', 50, TRUE)
 			var/turf/target = get_turf(A)
 			vortex(target,user)
+			addtimer(CALLBACK(src, /obj/item/singularityhammer/proc/recharge), 100)

```

Comment:
We generally use the `.proc/recharge` format on the same type.

Reply:
Gottcha